### ISSUE TITLE
Error(s) is not announced to a Screen Reader when saving a question

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2269]
![image](https://github.com/NationalSecurityAgency/skills-service/assets/51749745/429eaee7-0adf-4b47-a845-a9e6f3fa0877)

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/2272

**Comment by dwalizer** [comment]
:+1:
